Serverless enable add/set primary region (#17711)
cockroachdb/docs: src/current/_includes/releases/cloud/2023-08-22.md

<h3 id="2023-08-16-general-changes"> General changes </h3>

> - [Adding regions and changing the primary region]({% link cockroachcloud/folders.md %}) in the {{ site.data.products.cloud }} Console is now available for multi-region CockroachDB {{ site.data.products.serverless }} clusters. You cannot currently edit the region configuration for a single-region cluster once it has been created, and you cannot remove a region once it has been added. For multi-region CockroachDB {{ site.data.products.serverless }} guidance, refer to [Planning a CockroachDB {{ site.data.products.serverless }} Cluster]({% link cockroachcloud/plan-your-cluster-serverless.md %}#multi-region-clusters).

Reviewer: ```suggestion
- In the {{ site.data.products.cloud }} Console, you can now [add regions and change the primary region]({% link cockroachcloud/folders.md %}) for multi-region  CockroachDB {{ site.data.products.serverless }} clusters. You cannot currently edit the region configuration for a single-region cluster once it has been created, and you cannot remove a region once it has been added. For details, refer to [Planning a CockroachDB {{ site.data.products.serverless }} Cluster]({% link cockroachcloud/plan-your-cluster-serverless.md %}#multi-region-clusters).
```
